server/app/views/components/QuestionBank.java

```diff
@@ -219,50 +232,65 @@ private DivTag renderQuestionDefinition(QuestionDefinition definition) {
   }
 
   /**
-   * Used to filter questions in the question bank.
+   * Returns DOM attribute that is added on question DOM elements. If a question contains attribute
+   * - it means it is eligible for the given block and can be added to it.
+   */
+  public static String getQuestionEligibleForBlockAttribute(long blockId) {
+    return "data-block-" + blockId;
+  }
+
+  /**
+   * This functions used on for elements that trigger displaying question bank. Upon showing
+   * question bank has to know for which block it was triggered. That information passed through a
+   * data attribute on the button that triggered showing the bank. This method sets that attribute.
+   */
+  public static void setCurrentBlockAttribute(ButtonTag element, long blockId) {
+    element.attr(
+        QuestionBank.CURRENT_BLOCK_ID_ATTRIBUTE,
+        QuestionBank.getQuestionEligibleForBlockAttribute(blockId));
+  }
+
+  /**
+   * Used to check whether given question can be added to the given block.
```

Comment:
Summary fragments should be written as if they started with "This method" and should say what the method does. This content should come second.

https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment